Resources/config/services.xml

```diff
@@ -5,23 +5,16 @@
     <services>
         <!-- sulu-admin -->
         <service id="sulu_redirect.admin" class="Sulu\Bundle\RedirectBundle\Admin\RedirectAdmin" public="false">
+            <argument type="service" id="Sulu\Bundle\AdminBundle\Admin\Routing\RouteBuilderFactoryInterface"/>
```

Comment:
Actually I always already use it like that. Because this way it is easier to override it. If we want to change that it has to be done in the core as well. Any reason not to use it this way?

Reply:
Inside bundles we should always reference by ids and not by FCQN.